### PR TITLE
Redesign/partnerships page

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

Partnerships page redesign after usability testing.

This includes content changes and styling tweaks. There's also been a change to the partnership cards. "In flight" and "Past" partnerships have been removed and are all consolidated together.

[Figma](https://www.figma.com/file/LW1JsT0h3IY2trMHdIVDQc/CDS-Website-Revamp-(Work-in-Progress)?node-id=1616%3A11873)

### CODE REVIEW

**Comment by omartehsin1** [comment]
@Charcharbinkz could you just take a look at the content on the partnerships page and see if everything is good to go

**Comment by Charcharbinkz** [comment]
Thanks @omartehsin1 ! Looks so great. 

Would these minor changes be possible?

- Changing the subheadings of "Consultation" "Exploration" and "Embedded Team" from <p> to <h3> ? 
- Changing the subheadings of "Time commitment" and "How it works" from <p> to <h4> ? 
- The text and the button text in the "contact us" banner have changed slightly in both English and French from original

**Comment by Charcharbinkz** [comment]
Looks great! I've shared with Chelsea for last look, and I'll let SCMA know that we'll go live with this next Wednesday.

File: layouts/section/partnerships.html
Comment (dj2) [inline layouts/section/partnerships.html:17]: I didn't see any HTML in the various values, why do these all have `| safeHTML` on them?

File: layouts/section/partnerships.html
Comment (dj2) [inline layouts/section/partnerships.html:78]: Do we want to limit the number of things shown here in any way? It's currently at 14 cards I think?

File: layouts/section/partnerships.html
Comment (dj2) [inline layouts/section/partnerships.html:84]: This ends up causing an accessibility issue `Heading elements are not in a sequentially-descending order` as the partial starts at `h4` and the page last had an `h2` as the `h3` above was removed.

File: layouts/section/partnerships.html
Comment (omartehsin1) [reply layouts/section/partnerships.html:78]: The redesign has all the cards being displayed at once. So we're no longer splitting them up.

File: layouts/section/partnerships.html
Comment (omartehsin1) [reply layouts/section/partnerships.html]: Sorry, think it was lingering from before and never removed it. fixed now.

File: layouts/partials/partnerships_product_item.html
Comment (dj2) [inline layouts/partials/partnerships_product_item.html:9]: Can you check where this template is used? If it's used anywhere else we have to make sure this is still valid. (There are 2 of these templates with different headers if memory serves, maybe this allows consolidating to one?)

Comment (omartehsin1) [reply layouts/partials/partnerships_product_item.html:9]: It's only used once in `layouts/section/partnerships.html`. Since in that section I removed the `<h3>` tag and there is only an `<h2>` tag left, this looks like it satisfies the requirements of sequential headers.

**Comment by dj2** [comment]
LGTM from a dev perspective. Will leave approval for the website folks.

**Review by victoriachanimal — APPROVED** [review]
Yay, thank you @omartehsin1!

Content + contact link are great. To keep with timelines, good to merge if team is. 

After page goes live, will defer to @katedee when she's back for feedback on any follow up design tweaks (if needed).

**Review by Charcharbinkz — APPROVED** [review]
Looks great! Checked through links and triple checked the text. 

Looks good to merge from my perspective. Thanks @omartehsin1 :)

**Pull request merged**
